src/docs/product/alerts-notifications/notifications/index.mdx

```diff
@@ -60,39 +47,34 @@ Deploy emails are sent to users who have committed to the release that was deplo
 
 ## Quota
 
-Sentry sends all owners of the organization emails when
+Sentry sends all owners of the organization emails when:
 
 - The event volume is approaching or has exceeded the quota
 - On-demand is activated
 - Spike protection is activated or deactivated
 
-## **Notification Subscription**
+For more details, see the [full documentation on Quotas](/product/accounts/quotas/).
 
-You can configure your personal workflow and deploy notifications [in your account settings](https://sentry.io/settings/account/notifications/). You **cannot configure** quota notifications.
+## Notification Settings
 
-### **Workflow Notification Settings**
+You can configure your personal [workflow](#workflow) and [deploy](#deploy) notifications in your [account settings](https://sentry.io/settings/account/notifications/). You **cannot configure** quota notifications.
 
-![Radio buttons with options for always, ony on issues I subscribe to, and never.](workflow_notifications_subscription.png)
+### Workflow Notification Settings
 
-After selecting the appropriate alert setting, selectively change it by project in **User Settings > Account > Fine tune alerts by project**.
+Configure your notifications broadly for all notifications and/or according to project by navigating to **User Settings > Account**.
 
-Each project has three options: Default, On, or Off. Selecting default uses your default preference from the previous step.
+![Radio buttons with options for always, ony on issues I subscribe to, and never.](workflow_notifications_subscription.png)
 
-![A dropdown with the options to choose default, always, only on issues I subscribe to, and never.](fine_tune_workflow.png)
+After selecting the appropriate alert setting, selectively change it by project in **User Settings > Account > Fine tune alerts by project**. Selecting default uses your default preference from the previous step.
 
-**Unsubscribe**
+#### Unsubscribe
 
 To opt-out of workflow notifications for a specific issue, click "Unsubscribe" at the bottom right of the issue’s page.
 
 ![An unsubscribe button to stop notifications.](unsubscribe.png)
```

Comment:
This button has moved, it's near the top of the issue page now (lmk if you can't find it, good user test for us)